rfd/0080-yubikey-piv.md

```diff
@@ -0,0 +1,429 @@
+---
+authors: Brian Joerger (bjoerger@goteleport.com)
+state: draft
+---
+
+# RFD 80 - PIV smart card integration
+
+## Required approvers
+
+Engineering: @jakule && @r0mant
```

Comment:
Let's add @codingllama here as well as he's done a deep review of the RFD and has a lot of context.

Reply:
Happy to review.